package.json

```diff
@@ -18,13 +18,10 @@
 	"main": "index.js",
 	"jest": {
 		"collectCoverage": false,
-		"collectCoverageFrom": [
-			"<rootDir>/*.js"
-		],
 		"roots": [
 			"<rootDir>/test"
 		],
-		"testRegex": "\\.js"
```

Comment:
why the "test" prefix? i want jest to assume every file in a directory I pass it is a test.